### DISCUSSION TITLE
Accessibility: keyboard navigation is not visible on calendar

### DISCUSSION THREAD

**Discussion by ravisit** [primefaces/community#1524] [PrimeNG] [answered]
Keyboard navigation is not visible on calendar while NVDA tool is being used.
please find below attached two screen shot being observed at primeNg website.
![Screenshot (100)](https://github.com/primefaces/community/assets/29448663/01cf47a4-c321-499c-8bd1-2f154ba25fd5)
![Screenshot (99)](https://github.com/primefaces/community/assets/29448663/677274bd-dc5f-4591-a8b0-6661079383bd)

Please suggest solution for it.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Accessibility testing tools vary in diversity and they all have different qualifications. We cannot consider all the requests for these accessibility tools. Accessibility is implemented according to WCAG standards and keyboard navigation works properly in the calendar component.
